treeherder/model/models.py

```diff
@@ -616,6 +616,8 @@ class FailureLine(models.Model):
     id = BigAutoField(primary_key=True)
     job_guid = models.CharField(max_length=50)
     repository = models.ForeignKey(Repository)
+    job_log_id = models.BigIntegerField(null=True)  # This will become a foreign key once job_log is moved to this database
```

Comment:
How hard is this going to be to migrate to a ForeignKey (or FlexibleForeignKey, I guess?) relation when we do this? If it's going to be complicated, maybe we should just go ahead and migrate the job_log tables now.